tools/documentation_crawler/documentation_crawler/spiders/common/spiders.py

```diff
@@ -39,6 +39,11 @@
 
     # Warnings that are probably less important.
     r'The “type” attribute is unnecessary for JavaScript resources\.',
+
+    # nav is a section and thereby according to the HTML5 spec defines a scope of headings.
+    # The W3C example also shows an h1 tag in <nav>.
+    # https://html.spec.whatwg.org/multipage/sections.html#the-nav-element
+    r'Consider using the “h1” element as a top-level heading only \(all “h1” elements are treated as top-level headings by many screen readers and other tools\)\.',
```

Comment:
But the spec [also says](https://html.spec.whatwg.org/multipage/sections.html#headings-and-sections):

> Sections may contain headings of any rank, but authors are strongly encouraged to either use only `h1` elements, or to use elements of the appropriate rank for the section's nesting level.

And the validator is warning about the behavior of real accessibility tools, so I’m not convinced we should dismiss this warning merely on the grounds of what the spec allows.